sql/promscale--0.3.0--0.3.1.sql

```diff
@@ -0,0 +1,49 @@
+CREATE OR REPLACE FUNCTION @extschema@.label_find_key_equal(key_to_match prom_api.label_key, pat prom_api.pattern)
```

Comment:
1. Shouldn't we assert or set ownership of these functions? 
2. Does it matter whether a type was fully specified or not when aggregates and operators were defined? If it does, they may require a similar treatment.